packages/relay/tests/lib/eth/eth_call.spec.ts

```diff
@@ -770,6 +835,41 @@ describe('@ethCall Eth Call spec', async function () {
       expect(result).to.eq(EXAMPLE_CONTRACT_BYTECODE);
     });
 
+    it('should return null when blockParam is null in extractBlockNumberOrTag', async function () {
+      const result = await contractService['extractBlockNumberOrTag'](null, requestDetails);
```

Comment:
why calling it like this?

Reply:
`extractBlockNumberOrTag` is private and this is the pattern that we have been using so far in the rest of the tests